ui_framework/components/bar/bar.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export const KuiBar = ({ children, className, ...rest }) => {
+  const classes = classNames('kuiBar', className);
+  return <div className={classes} {...rest} >{children}</div>;
+};
+KuiBar.propTypes = {
+  children: React.PropTypes.node,
+  className: React.PropTypes.string,
```

Comment:
Can we import `PropTypes` and refer to that module here instead of using `React.PropTypes`?